### DISCUSSION TITLE
[Evaluations] Are all the finetunes stupid or is it just my methodology?

### DISCUSSION THREAD

**Discussion by Ph0rk0z** [oobabooga/text-generation-webui#1477] [Q&A]
I did a bunch of testing on some models at 512 context. The results are below. 

It appears you are better off using llama-7b than any of these vicuna or other models if testing them on wikitext means anything. I still have to try the 30Bs but they take 1/2 an hr to an hr to bench. It confirmed my theory that GPT-4chan is smarter than you think. Using the autograd code also doesn't make the models dumber, it scores identically.. but loras will not make your model smarter either.

Thoughts?

![Finetune_eval](https://user-images.githubusercontent.com/59298527/233778532-13df80b7-f452-411d-8d83-8f3c5dbe0bf8.png)

**Comment by BetaDoggo** [comment]
The wikitext benchmark isn't really designed to compare instruct models so this isn't that surprising. Finetuning makes models better at a specific task while making them less versatile overall. In your chart we can see this with alpaca-native-7b and vicuna-13b. For real world use vicuna follows instructions much better than alpaca-native yet it benchmarks slightly worse.

**Comment by nullgate** [comment]
The perplexity of SFT models should be worse than foundation model. There is no free lunch.
The purpose of SFT is NOT to reduce perplexity, but to improve alignment (e.g.: following instruction).

There will be alignment tax even for RLHF. Please read OpenAI's original paper for more details.
